openml/tasks/functions.py

```diff
@@ -306,21 +308,25 @@ def get_tasks(task_ids, download_data=True):
     ----------
     task_ids : iterable
         Integers/Strings representing task ids.
-    download_data : bool
+    download_data : bool (default = True)
         Option to trigger download of data along with the meta data.
+    download_qualities : bool (default=True)
+        Option to download 'qualities' meta-data in addition to the minimal dataset description.
 
     Returns
     -------
     list
     """
     tasks = []
     for task_id in task_ids:
-        tasks.append(get_task(task_id, download_data))
+        tasks.append(get_task(task_id, download_data, download_qualities))
     return tasks
 
 
 @openml.utils.thread_safe_if_oslo_installed
-def get_task(task_id: int, download_data: bool = True) -> OpenMLTask:
+def get_task(
+    task_id: int, download_data: bool = True, download_qualities: bool = True
```

Comment:
Above we allow `task_id` to be of type `str`, but here it's only int. How about we deprecate the use of anything besides `str` as I really don't see a reason to support it?